Source/ACE/Network/Enum/Spell.cs

```diff
@@ -8,10 +8,6347 @@ namespace ACE.Network.Enum
 {
     public enum Spell : uint
     {
-        FlameBolt           = 0x1B,
-        FrostBolt           = 0x1C,
-        LightningBolt       = 0x4B,
-        ForceBolt           = 0x56,
-        WhirlingBlade       = 0x5C,
+        Undef,
```

Comment:
Was it not possible to export these with values?  Just curious. Would have made finding stuff by value a lot easier.